.pre-commit-config.yaml

```diff
@@ -1,6 +1,13 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: stable
+-   repo: https://github.com/psf/black
```

Comment:
new black repo??

Reply:
Idk I think we were referring to a fork...

Reply:
Ah it moved from a developer's individual account to the python software foundation. The previous is an alias for the new correct name